charts/k8s-image-swapper/templates/deployment.yaml

```diff
@@ -85,6 +85,8 @@ spec:
               subPath: config.yaml
               mountPath: /.k8s-image-swapper.yaml
               readOnly: true
+            - mountPath: /tmp
+              name: tmp
```

Comment:
The functionality of caching the images seems to rely on being able to write to `/tmp`.  As it currently is, if `readOnlyRootFilesystem` is enabled in the security context then `/tmp` can't be written to.  Since `/tmp` is empty in the image it seemed like the best solution was to mount an emptyDir there to move it out of the root file system and allow it to be written to.